docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -240,6 +240,9 @@ quarkus.mongodb.connection-string = mongodb://localhost:27017
 
 If you need more configuration properties, there is a full list at the end of this guide.
 
+WARNING: By default Quarkus will restrict the use of JNDI within an application, as a precaution to try and mitigate any future vulnerabilities similar to log4shell.
+Because mongo+srv protocol often used on MongoDB connection string requires JNDI this protection will be automatically disabled.
```

Comment:
```suggestion
Because the `mongo+srv` protocol often used to connect to MongoDB requires JNDI, this protection is automatically disabled when using the MongoDB client extension.
```